content/en/docs/tasks/security/authorization/authz-ingress/index.md

```diff
@@ -31,143 +26,390 @@ through the Istio ingress gateway with this command:
 IP addresses not in the list will be denied. The `ipBlocks` supports both single IP address and CIDR notation.
-Create the authorization policy:
-
-    {{< text bash >}}
-    $ kubectl apply -f - <<EOF
-    apiVersion: security.istio.io/v1beta1
-    kind: AuthorizationPolicy
-    metadata:
-      name: ingress-policy
-      namespace: istio-system
-    spec:
-      selector:
-        matchLabels:
-          app: istio-ingressgateway
-      action: ALLOW
-      rules:
-      - from:
-        - source:
-           ipBlocks: ["1.2.3.4", "5.6.7.0/24"]
-    EOF
-    {{< /text >}}
 
-1. Verify that a request to the ingress gateway is denied:
+{{< tabset category-name="source" >}}
+
+{{< tab name="ipBlocks" category-value="ipBlocks" >}}
+
+Create the AuthorizationPolicy:
+
+{{< text bash >}}
+$ kubectl apply -f - <<EOF
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: ingress-policy
+  namespace: istio-system
+spec:
+  selector:
+    matchLabels:
+      app: istio-ingressgateway
+  action: ALLOW
+  rules:
+  - from:
+    - source:
+        ipBlocks: ["1.2.3.4", "5.6.7.0/24"]
+EOF
+{{< /text >}}
+
+{{< /tab >}}
+
+{{< tab name="remoteIpBlocks" category-value="remoteIpBlocks" >}}
+
+{{< text bash >}}
+$ kubectl apply -f - <<EOF
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: ingress-policy
+  namespace: istio-system
+spec:
+  selector:
+    matchLabels:
+      app: istio-ingressgateway
+  action: ALLOW
+  rules:
+  - from:
+    - source:
+        remoteIpBlocks: ["1.2.3.4", "5.6.7.0/24"]
+EOF
+{{< /text >}}
+
+{{< /tab >}}
+
+{{< /tabset >}}
+
+* Verify that a request to the ingress gateway is denied:
 
     {{< text bash >}}
     $ curl "$INGRESS_HOST":"$INGRESS_PORT"/headers -s -o /dev/null -w "%{http_code}\n"
     403
     {{< /text >}}
 
-1. Update the `ingress-policy` to include your client IP address:
-
-    {{< text bash >}}
-    $ kubectl apply -f - <<EOF
-    apiVersion: security.istio.io/v1beta1
-    kind: AuthorizationPolicy
-    metadata:
-      name: ingress-policy
-      namespace: istio-system
-    spec:
-      selector:
-        matchLabels:
-          app: istio-ingressgateway
-      action: ALLOW
-      rules:
-      - from:
-        - source:
-           ipBlocks: ["1.2.3.4", "5.6.7.0/24", "$CLIENT_IP"]
-    EOF
-    {{< /text >}}
-
-1. Verify that a request to the ingress gateway is allowed:
+* Update the `ingress-policy` to include your client IP address:
+
+{{< tabset category-name="source" >}}
+
+{{< tab name="ipBlocks" category-value="ipBlocks" >}}
+
+Find your original client IP address if you don't know it and assign it to a variable:
+
+{{< text bash >}}
+$ CLIENT_IP=$(kubectl get pods -n istio-system | grep ingress | awk '{print $1}' | while read -r pod; do kubectl logs "$pod" -n istio-system | grep remoteIP; done | head -1 | awk -F, '{print $3}' | awk -F: '{print $2}' | sed 's/ //') && echo "$CLIENT_IP"
```

Comment:
Same question as https://github.com/istio/istio.io/pull/8390/files#r516295115

Reply:
Same reasoning here; we can't be sure which ingress gateway got the request in the case of multiple ingress gateway pods.  We need to loop through them all and print the logs to find the request.